avocado/core/job.py

```diff
@@ -42,7 +42,7 @@
 from . import multiplexer
```

Comment:
Well based on the commit message and diff between the two files it looks like you changed several things at once, which is not really nice. You should fix style, in next commit add improvements and in other commit move things around, otherwise it's hard to make sure there were no typos/unintended changes.